visivo/query/insight_tokenizer.py

```diff
@@ -0,0 +1,499 @@
+import re
```

Comment:
This file will be very different, the query generator will be very different using SQL glot.